aries-site/src/examples/templates/forms/SettingsExample.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Main,
+  MaskedInput,
+  RangeInput,
+  Text,
+  TextInput,
+} from 'grommet';
+
+import { FormContainer } from '.';
+
+export const SettingsExample = () => {
+  const [formValues, setFormValues] = React.useState({
+    doNotDisturbHours: '9:00pm - 5:00am',
+    notifications: true,
+    notificationsVolume: 0.7,
+  });
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Heading level={3} margin="none">
+            Settings
+          </Heading>
+          <Text>for HPE Service</Text>
+        </Header>
+        <Main
+          // Padding used to prevent focus from being cutoff
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Form
+            value={formValues}
+            onChange={setFormValues}
+            onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+          >
+            <Box margin={{ bottom: 'small' }}>
+              <Text margin={{ left: 'small' }}>Key feature</Text>
+              <FormField
+                name="notifications"
+                label="Notifications"
+                component={CheckBox}
+                reverse
+                help="Description of feature"
+                toggle
+              />
+              <FormField
+                htmlFor="notificationsVolume"
+                name="notificationsVolume"
+                label="Notifications Volume"
+              >
+                <RangeInput
+                  id="notificationsVolume"
+                  name="notificationsVolume"
+                  max={1}
+                  min={0}
+                  pad
+                  step={0.1}
+                />
+              </FormField>
+            </Box>
+            <Box margin={{ bottom: 'small' }}>
+              <Text margin={{ left: 'small' }}>Key feature</Text>
+              <FormField
```

Comment:
In general, what should focus behavior be for form fields with border? Right now focus is appearing on the outer box and the checkbox itself. This is probably on the grommet side and shouldn't inhibit this PR but I think this keyboard behavior should be discussed.

<img width="361" alt="Screen Shot 2020-03-05 at 9 08 21 AM" src="https://user-images.githubusercontent.com/12522275/76006207-02639080-5ec1-11ea-9a55-86f27b2d8db6.png">

Reply:
I will add this point to: https://github.com/hpe-design/design-system/issues/458

Reply:
Added to: https://github.com/hpe-design/design-system/issues/458